### PR TITLE
Initialize MDFTesting pod with examples.

### PR DESCRIPTION
**Pull request by fluxminds**
This PR is related to https://github.com/material-components/material-components-ios/pull/8435, from which the code logic is pulled.

### CODE REVIEW

**Review by jverkoey — CHANGES_REQUESTED** [review]
You have some personal files checked in to this PR; please double-check your .gitignore and re-upload the PR.

**Comment by fluxminds** [comment]
Sorry. I think I committed some files before adding gitignore. Re-uploaded the changes.

**Review by jverkoey — COMMENTED** [review]
I see you're using Swift package manager's `Sources/` convention for the paths here. Can we align on our team's conventions of a `src` directory instead?

Swift package manager added support for custom paths in https://github.com/apple/swift-evolution/blob/master/proposals/0162-package-manager-custom-target-layouts.md, so I'd prefer we keep a self-consistent file organizational structure across all of our projects.

**Comment by fluxminds** [comment]
Definitely! It is addressed in f1a650a9af40d5b24f0d69995a07fd69c447a26e.

File: Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//
```
Comment (jverkoey) [inline Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m]: Similarly here: `examples/`.

File: Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+//  MDFTestingExampleTests.m
+//  MDFTestingExampleTests
+//
+//  Created by Wenyu Zhang on 9/24/19.
+//  Copyright © 2019 Wenyu Zhang. All rights reserved.
+//
```
Comment (jverkoey) [inline Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m]: Please update these copyright headers with the apache licenses.

**Review by jverkoey — APPROVED** [review]

File: Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m (hunk 1 above)
Comment (fluxminds) [reply Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m]: Addressed in 030297b and 790d03f.

File: Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m (hunk 2 above)
Comment (fluxminds) [reply Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m]: Addressed in b5f238d2c4d4f1be6fe1881df69fc5b91ed1e6c2.

**Pull request merged**